### PR TITLE
Add GA_init

### PR DESCRIPTION
**Pull request by stevenroose**

### CODE REVIEW

**Comment by shesek** [comment]
Is this used by the Green apps, or meant for testing?

**Comment by stevenroose** [comment]
I was just going over the gdk.h interface looking for documentation on GA_connect, GA_create_session etc and I noticed it.

**Comment by shesek** [comment]
Interesting. The Green android app somehow managed to work without this :)

File: src/lib.rs [hunk 1]
```diff
@@ -161,6 +161,16 @@ pub extern "C" fn GA_get_networks(ret: *mut *const GA_json) -> i32 {
 #[cfg(feature = "android_logger")]
 static INIT_LOGGER: Once = ONCE_INIT;
 
+#[no_mangle]
+pub extern "C" fn GA_init(_config: *const GA_json) -> i32 {
+    debug!("GA_init()");
```
Comment (shesek) [inline src/lib.rs]: Maybe log the config object passed in, for debugging purposes?

Comment (stevenroose) [reply src/lib.rs]: Good idea. Done.

**Pull request merged**